src/components/RegisterPage.js

```diff
@@ -124,12 +124,12 @@ export default (props) => {
 
     cleanErr()
 
-    doRegPage.Register(myID, username, password, passwordConfirm, email, over18)
+    doRegPage.Register(myID, username, password, passwordConfirm, email, over18, veriCode)
   }
 
   let verifyEmail = () => {
-    document.getElementById("emailField").readOnly = true
```

Comment:
I added this just in order to prevent users from editing the email again after getting the verification code.
(Of course I should have use the React way :P. Sorry)
Or will this action be blocked in backend so we don't need to handle this here?

Reply:
Thanks for reminding me this issue～

1. We can do it in react-way in frontend.

2. We should also check the match in middleware, or someone else may just use curl to fake the api-call.
    I should store both token and email in the redis in attemptregister and verify that both token and email should match in /account/register.

Reply:
Updated version is that the email is readOnly when isSetVerifyEmail.

Reply:
Checked! thank you for your reply~